modules/standard/Random.chpl

```diff
@@ -130,6 +150,15 @@ module Random {
        with at least 48 mantissa bits for `real(64)` data.
 
 
+     The values generated by this NPB RNG do not include 0.0 and 1.0.
```

Comment:
Yeah, I thought that at first, but then looked closely at the comment we pasted from the NPB reference. It use (0,1).

Remember that we're forcing the seed to be odd? The multiplier is also odd.
So, the NPB algorithm starts with an odd number, multiplies it by an odd number, and then mods it by a power of 2 (which keeps it odd). For the result of the mod to be 0, the argument has to be a multiple of the power of two, which means it has to be even. Which can never happen.

Here's another argument. Since the NPB RNG multiplies by `a` and then does a mod operation, if its output were ever 0, that would imply that all future outputs would be 0 (since multiplying 0 by a just yields 0, and modding changes nothing). So, this RNG would be *really bad* if it ever generated a 0.